spec/lib/infra_conversion_throttler_spec.rb

```diff
@@ -25,18 +25,20 @@
       allow(ems).to receive(:conversion_hosts).and_return([conversion_host1, conversion_host2])
       allow(conversion_host1).to receive(:check_ssh_connection).and_return(true)
       allow(conversion_host2).to receive(:check_ssh_connection).and_return(true)
+      allow(conversion_host1).to receive(:authentication_check).and_return([true, 'passed'])
+      allow(conversion_host2).to receive(:authentication_check).and_return([true, 'passed'])
```

Comment:
keep this

but this stubbing is skipping actually testing the code.
try and keep stubbing to a method that provides just an external interface (e.g.: connect to ssh - or maybe even connect to the database, though that is suspect)

Reply:
Those particular tests are just checking the throttling, not sure what you mean.